src/coreComponents/physicsSolvers/multiphysics/SinglePhasePoromechanicsFluxKernels.cpp

```diff
@@ -505,7 +505,7 @@ void FaceElementFluxKernel::
   typename SurfaceElementStencilWrapper::IndexContainerViewConstType const & sesri = stencilWrapper.getElementSubRegionIndices();
   typename SurfaceElementStencilWrapper::IndexContainerViewConstType const & sei = stencilWrapper.getElementIndices();
 
-  forAll< parallelDevicePolicy<> >( stencilWrapper.size(), [=] GEOSX_HOST_DEVICE ( localIndex const iconn )
+  forAll< serialPolicy >( stencilWrapper.size(), [=] GEOSX_HOST_DEVICE ( localIndex const iconn )
```

Comment:
Why is serialPolicy used instead of parallelDevicePolicy?

Reply:
coz I had forgotten to change it back